app/views/episodes/index.html.erb

```diff
@@ -5,6 +5,7 @@
     <h1 class="text-black fw-bold d-flex flex-fill my-4"><%= t(".episodes") %></h1>
     <div class="d-flex gap-2 align-items-center">
       <% if @podcast.present? %>
+        <% if policy(@podcast).update? %>
```

Comment:
This could be `policy(@podcast.episodes.new).create?` ... in case we someday want to give some users edit permission but not create.